packages/asset-service/src/service/AssetService.ts

```diff
@@ -29,14 +39,14 @@ export class AssetService {
     if (description) return { description, isTrusted: true }
```

Comment:
need to add a new parameter to the description method called `locale` of type `LocaleType`.

Also with the new structure of the json, need to update the descriptions type to `Record<string, Record<LocaleType, string>>` and need to change line 36 to

`const description = descriptions[assetId]?.[locale] || descriptions[assetId]?.en || ''`

Reply:
Done.

Reply:
Is it? There is no tests update in this diff